Userland/Shell/Tests/builtin-redir.sh

```diff
@@ -2,9 +2,9 @@
 
 source $(dirname "$0")/test-commons.inc
 
-rm -rf shell-test
-mkdir -p shell-test
-cd shell-test
+rm -rf /tmp/shell-test 2> /dev/null
```

Comment:
does this actually have some sort of output?

Reply:
Not sure. Monkey see, monkey do, that's what some of the other tests had. I can omit it. I would expect -f to not care if the folder doesn't exist.

Reply:
Sure, that's fine I guess